### ISSUE TITLE
Bug: Patcher crashing on darwin platform

### ISSUE THREAD

**Issue by MidKnightXI** [MidKnightXI/opgg-ads-remover#9]
Tested recently the patcher on my mac and this happened:

```
Killing OPGG process
Unpacking OPGG asar file
node:fs:585
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open '/Applications/OP.GG.app/Contents/Resources/app.asar.unpacked/node_modules/gifsicle/cli.js'
    at Object.openSync (node:fs:585:3)
    at Object.openSync (pkg/prelude/bootstrap.js:796:32)
    at Object.readFileSync (node:fs:453:35)
    at Object.readFileSync (pkg/prelude/bootstrap.js:1082:36)
    at Object.readFileSync (/snapshot/opgg-remover/node_modules/asar/lib/disk.js:110:17)
    at Object.extractAll (/snapshot/opgg-remover/node_modules/asar/lib/asar.js:204:28)
    at rebuildAddDir (/snapshot/opgg-remover/bundle.js)
    at main (/snapshot/opgg-remover/bundle.js)
    at Object.<anonymous> (/snapshot/opgg-remover/bundle.js)
    at Module._compile (pkg/prelude/bootstrap.js:1930:22) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/Applications/OP.GG.app/Contents/Resources/app.asar.unpacked/node_modules/gifsicle/cli.js'
}
```

will look into that later

**Comment by MidKnightXI** [comment]
Apparently version 3.2 of the asar lib was causing the crash, it's now fixed!

**Comment by MidKnightXI** [comment]
Fixed in v1.2
